libraries/mbed/hal/sleep_api.h

```diff
@@ -55,6 +55,22 @@ void sleep(void);
  */
 void deepsleep(void);
 
+
+/** Send the microcontroller to ultra deep sleep (VLLS0 with POR disabled)
```

Comment:
VLLS0 is Kinetis feature, therefore should be not here but placed in the kinetis implementation.